tests/Unit/Listeners/SendVoucherHistoryEmailTest.php

```diff
@@ -42,23 +43,23 @@ protected function setUp()
 
         // Set up voucher states.
         Auth::login($this->user);
+
         foreach ($this->vouchers as $v) {
             $v->applyTransition('order');
             $v->applyTransition('print');
             $v->applyTransition('dispatch');
             $v->applyTransition('allocate');
             $v->trader_id = 1;
             $v->applyTransition('collect');
+            $v->applyTransition('confirm');
```

Comment:
oh, I see. If we need to take into account that we're selecting confirmed ones we could take the vouchers from the `$trader->vouchersConfirmed` list in the test too then?